EXT_SpeciesType.md

```diff
@@ -0,0 +1,90 @@
+Convention for Specifying Particle Species 
+==========================================
+
+openPMD extension name: `SpeciesType`
+
+openPMD extension ID: `4`
```

Comment:
note: this will be removed in `2.0.0` but we have to assign it since we are still in `1.X`